### PR TITLE
DM-1818 Update footer and header

### PR DESCRIPTION
**Pull request by emyl3**
### JIRA issue link
[DM-1818](https://agile6.atlassian.net/browse/DM-1818)

## Description - what does this code do?
This PR updates the header and footer to reflect the new designs and is mobile friendly.

## Testing done - how did you test it/steps on how can another person can test it 
Test the Header and Footer in different views (mobile, tablet, desktop) and on different browsers.
- Make sure the links redirect/work correctly
- The hover states of links are correct
- Return to top works
- Search still works
- You can now search for 1 character

## Link to mock-ups/mock ups (image file if you have it) (if applicable)
[Figma](https://www.figma.com/file/FogT5mBUVl8rE7OgWDQWeJ/VADM?node-id=276%3A0)

## Acceptance criteria
- [X] Move Partner link to footer
- [x] Add “Search” bar to header
- [X] Update header color scheme
- [X] Header should now be “Profile icon,” “Search,” “Shark Tank (starting thu 6/11)” “COVID-19”
- [X] Confirm with designs (I did this from memory)
- [X] Write unit tests
- [X] Release to STG and test

## Definition of done
- [ ] Unit tests written (if applicable)
- [ ] e2e/accessibility tests written (if applicable)
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating JIRA issue
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by ahay-agile6** [comment]
sign out button is missing from mobile menu
image

**Comment by emyl3** [comment]
> sign out button is missing from mobile menu
> image

For the sign out on mobile view, I believe Becca wanted it so that a user had to click "Your Profile" and then will see a dropdown of options (including "Sign Out")

**Comment by ahay-agile6** [comment]
got it. ok, thanks :)

**Comment by ahay-agile6** [comment]
i think that just shows how it doesn't look like the "Your Profile" button will do that, heh

**Comment by emyl3** [comment]
> i think that just shows how it doesn't look like the "Your Profile" button will do that, heh

I'll check in with Becca if I should add the plus sign back like how USWDS has (https://designsystem.digital.gov/design-principles/) Her designs didn't have them but yeah totally can be unclear! 😅

File: app/views/practices/_search.js.erb [hunk 1]
```diff
@@ -17,14 +17,16 @@ function searchPracticesPage() {
...
 
+        searchField.value = query;
         // Run the search query
         results = fuse.search(query);
 
         console.log(results.length)
 
         // Print the number of results for the query
-        resultsSummary.innerHTML = results.length + ' result' + (results.length === 1 ? '' : 's') + ' for "' + query + '":';
+        resultsSummary.innerHTML += `${results.length} ${results.length === 1 ? 'result' : 'results'} for <span class="text-bold">${query}</span>:`;
```
Comment (ahay-agile6) [inline app/views/practices/_search.js.erb:29]: we cannot use es6 in these types of files. only in the `.es6` files

**Pull request merged**